### PR TITLE
CASSANDRA-15679 Test COPY on collections with blobs

### PR DESCRIPTION
**Pull request by Gerrrr**
https://issues.apache.org/jira/browse/CASSANDRA-15679

### CODE REVIEW

File: cqlsh_tests/test_cqlsh_copy.py [hunk 1]
```diff
@@ -1788,9 +1794,15 @@ def test_all_datatypes_read(self):
...
-            data_set[2] = '0x{}'.format(''.join('%02x' % c for c in self.data[2]))
+            data_set[2] = _format_blob(self.data[2])
+            data_set[25][3] =  _format_blob(self.data[25][3])
+            data_set[26] = [_format_blob(el) for el in self.data[26]]
+            data_set[27] = {_format_blob(el) for el in self.data[27]}
             writer.writerow(data_set)
 
+        def _format_blob(blob):
+            return '0x{}'.format(''.join('%02x' % c for c in self.data[2]))
+
```
Comment (stef1927) [inline cqlsh_tests/test_cqlsh_copy.py:1805]: I'm getting an error at line 1797 unless I move `format_blob` before it gets used. Also, `self.data[2]` should be `blob`? Once I fixed this, there was a problem with the indexes for `data_set`. Is this what you meant: https://github.com/stef1927/cassandra-dtest-1/commit/966e6db437cd0a6ecd731ffe9a6279a275065341?

If that commit is correct, I still see problems in copyutil unfortunately.

File: cqlsh_tests/test_cqlsh_copy.py [hunk 2]
```diff
@@ -237,7 +237,10 @@ def all_datatypes_prepare(self):
                 u frozen<list<list<address_type>>>,
                 v frozen<map<map<int,int>,set<text>>>,
                 w frozen<set<set<inet>>>,
-                x map<text, frozen<list<text>>>
+                x map<text, frozen<list<text>>>,
+                y map<int, blob>,
+                z list<blob>,
+                aa set<blob>
```
Comment (stef1927) [inline cqlsh_tests/test_cqlsh_copy.py:243]: The columns need to be in alphabetical order, see 449e96b2a7aa8cc6dbbcf189e75b6f547e8e314f

**Comment by stef1927** [comment]
Committed manually as 88cc70b4d6b598c60d4b567cdceff49a76140312.

@Gerrrr would you mind double checking and closing the PR if you can (I can't)

**Comment by Gerrrr** [comment]
Thank you, @stef1927!

I double checked https://github.com/apache/cassandra-dtest/commit/88cc70b4d6b598c60d4b567cdceff49a76140312 and it looks correct; closing this PR.

**Pull request closed without merging**